extensions/core/deployment/src/main/java/org/apache/camel/quarkus/core/deployment/CamelSupport.java

```diff
@@ -117,18 +117,21 @@ public static boolean isPublic(ClassInfo ci) {
      * services from resources belonging to META-INF/services/org/apache/camel.
      */
     public static class ServiceInfo {
+        public final Path path;
```

Comment:
A piece of JavaDoc stating what Path is this would be nice. Something like `The path from which this {@link ServiceInfo} was read. E.g {@code META-INF/services/org/apache/camel/my-service}`